packages/tools/kadena-cli/src/account/utils/accountHelpers.ts

```diff
@@ -0,0 +1,52 @@
+import yaml from 'js-yaml';
+import { readdirSync } from 'node:fs';
+import { join } from 'node:path';
+import { z } from 'zod';
+
+import { ACCOUNT_DIR } from '../../constants/config.js';
+import { services } from '../../services/index.js';
+import { isEmpty } from '../utils/addHelpers.js';
+
+interface IAccount {
+  name: string;
+  alias: string;
+}
+
+const accountAliasFileSchema = z.object({
+  name: z.string(),
+  fungible: z.string(),
+  publicKeys: z.array(z.string()),
+  predicate: z.string(),
+});
+
+const readAccountFromFile = async (accountFile: string): Promise<IAccount> => {
+  const accountAlias = accountFile.split('.')[0];
+  const content = await services.filesystem.readFile(
+    join(ACCOUNT_DIR, accountFile),
+  );
+  const account = content !== null ? yaml.load(content) : null;
```

Comment:
When account is null, don't you want to throw?
accountAliasFileSchema.parse(account)) does not handle null gracefully, so either it throws an error upon encountering null or it doesn't return an object with a name property, then trying to access .name on the result will lead to a runtime error because you're trying to access a property on null

Reply:
zod parse is also taking care of null gracefully, but yeah it's not providing meaningful error message to the user. so wrapped it with `try` and `catch` like Bart suggested.